src/hooks/useViewTabs.ts

```diff
@@ -40,7 +40,7 @@ function mapStateToProps(store: Store): UseViewTabsState {
  * @param depth 1 for top level navigation; 2, 3, 4 for SecondLevelMenu, ThirdLevelMenu and FourthLevelMenu
  * @category Hooks
  */
-export function useViewTabs(depth: NavigationLevel) {
+export function useViewTabs(depth: NavigationLevel | number) {
```

Comment:
There is no point in this union: the resulting type will be extended to `number`. If all numbers now supported, `NavigationLevel` should be removed

Reply:
removed